packages/dynamite/dynamite/lib/src/helpers/default_value.dart

```diff
@@ -0,0 +1,17 @@
+import 'dart:convert';
+
+import 'package:built_value/json_object.dart';
+
+/// Encodes the given value to be used for generation.
+String? encodeDefault(JsonObject? $default) {
+  if ($default == null) {
+    return null;
+  }
+
+  var value = jsonEncode($default.value);
+  if ($default.isMap || $default.isList) {
```

Comment:
Set should also be const, no?

Reply:
$default is a `JsonObject` thus it can not be a set.